app/uk/gov/hmrc/helptosavefrontend/connectors/EmailVerificationConnector.scala

```diff
@@ -43,34 +43,33 @@ trait EmailVerificationConnector {
 @Singleton
 class EmailVerificationConnectorImpl @Inject()(http: WSHttp, conf: Configuration) extends EmailVerificationConnector with ServicesConfig with Logging {
 
-  val linkTTLMinutes = conf.underlying.getInt("services.email-verification.linkTTLMinutes")
+  val linkTTLMinutes = conf.underlying.getInt("microservice.services.email-verification.linkTTLMinutes")
   val emailVerifyBaseURL = baseUrl("email-verification")
   val verifyEmailURL = s"$emailVerifyBaseURL/email-verification/verification-requests"
 
   def isVerifiedURL(email: String) = s"$emailVerifyBaseURL/email-verification/verified-email-addresses/$email"
-
-  val continueURL = "email-verification.continue.baseUrl"
-  val templateId = "hts_email_verification"
+  val continueURL =  conf.underlying.getString("microservice.services.email-verification.continue-url")
+  val templateId = "awrs_email_verification"
 
   def verifyEmail(nino: String, newEmail: String)(implicit hc: HeaderCarrier): Future[Either[VerifyEmailError, Unit]] = {
-    val verificationRequest = EmailVerificationRequest(newEmail, nino, templateId, Duration.ofMinutes(linkTTLMinutes).toString, continueURL, Map())
-
+    val continueUrlWithParams = continueURL + "?p=" + EmailVerificationParams(nino, newEmail).encode()
+    val verificationRequest = EmailVerificationRequest(newEmail, nino, templateId, Duration.ofMinutes(linkTTLMinutes).toString, continueUrlWithParams, Map())
     http.post(verifyEmailURL, verificationRequest).map { (response: HttpResponse) ⇒
       response.status match {
         case OK | CREATED =>
           logger.info("[EmailVerification] - Email verification successfully triggered")
           Right(())
         case BAD_REQUEST ⇒
           logger.warn("[EmailVerification] - Bad Request from email verification service")
-          Left(RequestNotValidError(nino))
+          Left(RequestNotValidError())
         case CONFLICT ⇒
-          logger.info("[EmailVerification] - Email already verified")
-          Left(AlreadyVerified(nino, newEmail))
+          logger.info(s"[EmailVerification] - Email: $newEmail already verified")
```

Comment:
I think we do have to treat both emails and ninos as sensitive data - but also that we should be logging at this point. Dumping stuff into the audit logs just because it is sensitive is frowned upon, so we have a bit of an issue here. Could we make a call to the service to record the situation?